src/ServicePulse.Host/vue/src/components/failedmessages/AllDeletedMessages.vue

```diff
@@ -57,18 +61,13 @@ function loadMessages() {
   return loadPagedMessages(groupId.value, pageNumber.value, "", "", startDate.toISOString(), endDate.toISOString());
 }
 
-async function loadGroupDetails(groupId) {
+async function loadGroupDetails(groupId: string | string[]) {
```

Comment:
This function should not be taking a `string[]`, because we use that for string concatenation, which will produce something like `string[]`

Reply:
fixed